clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -192,21 +194,33 @@ export default async function handler(
   if (!fidesJS || fidesJS === "") {
     throw new Error("Unable to load latest fides.js script from server!");
   }
+  let fidesGPP: string = "";
+  if (gppEnabled) {
+    // eslint-disable-next-line no-console
+    console.log("GPP extension enabled, bundling fides-ext-gpp.js...");
```

Comment:
hm, this file is pretty inconsistent with whether / when we should console.log. Sometimes we wrap in `if (environment.settings.DEBUG) {}` and other times not. But since these are server-side, I'm not too concerned

Reply:
I thought I had updated that already here. It may have gotten lost in a revert I did or something. Good callout.

Reply:
oh! I updated all of these to be consistent in the next story/branch/PR. Look for this update there. It was also bugging me :)

Reply:
oh excellent!